pyvista/plotting/colors.py

```diff
@@ -165,6 +165,35 @@
 
 """
 
+from __future__ import (
+    annotations,  # Necessary for autodoc_type_aliases to recognize the 'color_like' alias
+)
```

Comment:
1: Yes
2: No, at least I don't think so, as the code worked fine without this on my machine (Windows, python 3.7). It is however needed for the sphinx autodoc extension to recognize the `color_like` type (see https://github.com/pyvista/pyvista/pull/2175#issuecomment-1035338661)

Yes, that's also why I think it's hard to move it to the `_typing` module?

Reply:
During my investigation that led me to `typing.TYPE_CHECKING` the penny dropped for me. The code worked fine because string-valued annotations are already lazy. We only need the `__future__` import to make non-string-valued annotations behave like string valued. My point is that if we keep the `__future__` import we can probably use `Color` instead of `'Color'` in the definition of `color_like`.

Reply:
Just a happy heads-up that https://github.com/pyvista/pyvista/pull/1716 will probably be merged soon, so >=3.7 is good to go.

Reply:
I moved it to the `_typing` module in latest commit.